eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Rules/TestChain.hs

```diff
@@ -630,8 +612,8 @@ potsSumIncreaseByRewardsPerTx SourceSignalTarget {source = chainSt, signal = blo
         } =
         (coinBalance u' <+> d' <+> f')
           <-> (coinBalance u <+> d <+> f)
-          === fold (UM.unUnify (UM.Rewards umap1))
-          <-> fold (UM.unUnify (UM.Rewards umap2))
+          === (UM.fromCompact (fold (UM.unUnify (UM.Rewards umap1))))
+          <-> (UM.fromCompact (fold (UM.unUnify (UM.Rewards umap2))))
```

Comment:
It's easy to see redundant parenthesis, if there is no infix operator inside the parenthesis, it is guaranteed to redundant.
```suggestion
          === UM.fromCompact (fold (UM.unUnify (UM.Rewards umap1)))
          <-> UM.fromCompact (fold (UM.unUnify (UM.Rewards umap2)))
```

Reply:
done